pallets/crowdloans/src/lib.rs

```diff
@@ -635,6 +646,31 @@ pub mod pallet {
             xcm_weight_misc: XcmWeightMisc<Weight>,
         ) -> DispatchResultWithPostInfo {
             T::UpdateOrigin::ensure_origin(origin)?;
+
+            let XcmWeightMisc {
```

Comment:
This may shouldn't be in Call, can you make it a member function for `XcmWeightMisc`